.github/workflows/test.yml

```diff
@@ -16,7 +16,15 @@ on:
 
 jobs:
   tests:
-    runs-on: buildjet-8vcpu-ubuntu-2204-arm
+    strategy:
+      matrix:
+        include:
+          - platform: linux/amd64
```

Comment:
Do we actually use this var anywhere?

Reply:
No.. Just put it in for readability. Can remove if we want.